widget/components/Sidebar.jsx

```diff
@@ -21,31 +21,75 @@ Object.keys(documents).forEach((path) => {
   }
 });
 
+const Sidebar = styled.div`
```

Comment:
I think for the ability to customize styles in other projects, for a white label solution, this CSS should be moved to `index`, where "sidebar" className was defined before